content/blog/accessing-hpe-data-fabric-s3-storage-from-spring-boot-s3-micro-service-deployed-in-k3s-cluster.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Accessing HPE Data Fabric S3 Storage from Spring Boot S3 Micro Service
+  deployed in K3S cluster
+date: 2021-09-13T08:13:40.734Z
+featuredBlog: true
+priority: 0
+author: Kiran Kumar Mavatoor
+authorimage: /img/hpe-dev-grommet-gremlin-rockin.svg
+tags:
+  - K3S
+  - Spring Boot
+  - HPE Data Fabric S3 Storage
+---
+S3 is a storage service offered by HPE Data Fabric Platform that offers scalable, secure, and well performing object storage. In this article, we will discuss as how to access HPE Data Fabric Object Store (S3) using Spring Boot S3 Micro Service application deployed in [K3S cluster](https://k3s.io/) and perform basic S3 operations like upload, list, delete etc. Below diagram gives the architecture overview 
```

Comment:
I thought S3 with a Amazon storage service. Maybe we should clarify, that this is a S3-compatible service and what is S3 initially.

Reply:
HPE Data Fabric is building Native S3 storage service. It will be compatible with AWS S3 SDK